source/guide/3.2/reference-debugging.md

```diff
@@ -9,6 +9,24 @@ pageord: 400
 
 {%summary%} Cloudify is very pluggable and extendible by its nature; This page explains how to debug custom plugins, workflows and other extensions or components of Cloudify. {%endsummary%}
 
+# Where Are My Log Files?
+
+Cloudify log files commonly reside on the manager host machine, in a Docker container.
+In order to access the container to view log files, first ssh into the host machine. From it, access the container
+using this command: `sudo docker exec -it cfy bash`
+
+# Security Audit Log
+
+When starting Cloudify manager in a secured mode requests sent to the REST server go through authentication. <br>
+The security audit log file documents successful and failed REST calls, their origin, method of authentication and
+the reason of failure, if authentication failed.
```

Comment:
i think english wise its better to say "method of authentication, and, if the authentication failed, the reason for failure".